### PR TITLE
chore: move skip responsibility to the automation, not the CI

### PR DESCRIPTION
**Pull request by mdelapenya**
## What does this PR do?
It moves the ability to skip scenarios tagged as `@skip` from the CI to the automation.

## Why is it important?
Now it's possible to decide whether to skip scenarios or not, setting it to true (do skip) by default.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the Unit tests for the CLI, and they are passing locally
- [x] I have run the End-2-End tests for the suite I'm working on, and they are passing locally
- [ ] I have noticed new Go dependencies (run `make notice` in the proper directory)

## Author's Checklist

- [ ] Variable expansion in Makefile

## How to test this PR locally

Do not skip scenarios:
```shell
$ SUITE="metricbeat" \
    SKIP_SCENARIOS=false \
    TAGS="integrations && apache" DEVELOPER_MODE=true \
    TIMEOUT_FACTOR=3 LOG_LEVEL=TRACE \
    make -C e2e functional-test

go get -v github.com/cucumber/godog/cmd/godog@v0.10.0
go: found github.com/cucumber/godog/cmd/godog in github.com/cucumber/godog v0.10.0
cd _suites/metricbeat && \
	OP_LOG_LEVEL=TRACE \
	OP_LOG_INCLUDE_TIMESTAMP=TRUE \
	TIMEOUT_FACTOR=3 \
	STACK_VERSION= \
	DEVELOPER_MODE=true \
	godog --format=pretty --tags "integrations && apache"
```

Skip scenarios (default):
```shell
$ SUITE="metricbeat" \
    TAGS="integrations && apache" DEVELOPER_MODE=true \
    TIMEOUT_FACTOR=3 LOG_LEVEL=TRACE \
    make -C e2e functional-test

go get -v github.com/cucumber/godog/cmd/godog@v0.10.0
go: found github.com/cucumber/godog/cmd/godog in github.com/cucumber/godog v0.10.0
cd _suites/metricbeat && \
	OP_LOG_LEVEL=TRACE \
	OP_LOG_INCLUDE_TIMESTAMP=TRUE \
	TIMEOUT_FACTOR=3 \
	STACK_VERSION= \
	DEVELOPER_MODE=true \
	godog --format=pretty --tags "integrations && apache && ~skip"
```

## Related issues

- Relates #439

### CODE REVIEW

**Comment by mdelapenya** [comment]
The CI logs reflect the changes are positive. Will be able to merge if the CI is green

**Comment by mdelapenya** [comment]
I ran the CI job setting the new Jenkins input parameter to `false`, so it won't skip the desired scenarios, causing the two errors (the scenario is skipped).

I'm gonna trigger the CI for the PR again, and if it passes, will merge.

**Comment by mdelapenya** [comment]
jenkins run the tests please

File: e2e/Makefile [hunk 1]
```diff
@@ -5,12 +5,18 @@ FORMAT?=pretty
...
+SKIP_SCENARIOS?=true
 STACK_VERSION?=
 PICKLES_VERSION?="2.20.1"
 VERSION_VALUE=`cat ../cli/VERSION.txt`
 
 ifneq ($(TAGS),)
 TAGS_FLAG=--tags
+ifeq ($(SKIP_SCENARIOS),true)
+## We always want to skip scenarios tagged with @skip
+TAGS+= && ~skip
```
Comment (mdelapenya) [inline e2e/Makefile:18]: I quoted the value here, but it ended up adding them to the value instead

**Review by cachedout — APPROVED** [review]

**Pull request merged**